waku/src/node/mod.rs

```diff
@@ -59,6 +66,34 @@ impl WakuNodeHandle<Running> {
     pub fn stop(self) -> Result<()> {
         stop_node()
     }
+
+    pub fn connect_peer_with_address(
+        &mut self,
+        address: Multiaddr,
+        timeout: Option<Duration>,
+    ) -> Result<()> {
+        peers::waku_connect_peer_with_address(address, timeout)
+    }
+
+    pub fn connect_peer_with_id(
+        &mut self,
+        peer_id: PeerId,
+        timeout: Option<Duration>,
+    ) -> Result<()> {
+        peers::waku_connect_peer_with_id(peer_id, timeout)
+    }
+
+    pub fn disconnect_peer_with_id(&mut self, peer_id: PeerId) -> Result<()> {
+        peers::waku_disconnect_peer_with_id(peer_id)
+    }
+
+    pub fn peer_count(&self) -> Result<usize> {
+        peers::waku_peer_count()
+    }
+
+    pub fn peers(&self) -> Result<WakuPeers> {
+        peers::waku_peers()
+    }
```

Comment:
I'm not sure if this methods need of the waku node to be `started`, or just `initialized would be enough`. Any hint @richard-ramos ?